lineman/app/services/ability_service.coffee

```diff
@@ -66,5 +66,9 @@ angular.module('loomioApp').factory 'AbilityService', (CurrentUser) ->
       (!membership.admin or membership.group().adminIds().length > 1) and
       (membership.user() == CurrentUser or @canAdministerGroup(membership.group()))
 
+    canDeactivateUser: ->
+     _.all CurrentUser.memberships(), (membership) ->
+       !membership.admin or membership.group().adminIds().length > 1
```

Comment:
Yeah, that'd be much better!

On Wed, Jul 8, 2015 at 4:15 AM, Robert Guthrie <notifications@github.com>
wrote:

> In lineman/app/services/ability_service.coffee
> <https://github.com/loomio/loomio/pull/2282#discussion_r34137079>:
>
> > @@ -66,5 +66,9 @@ angular.module('loomioApp').factory 'AbilityService', (CurrentUser) ->
> >        (!membership.admin or membership.group().adminIds().length > 1) and
> >        (membership.user() == CurrentUser or @canAdministerGroup(membership.group()))
> >
> > +    canDeactivateUser: ->
> > +     _.all CurrentUser.memberships(), (membership) ->
> > +       !membership.admin or membership.group().adminIds().length > 1
>
> aren't you going to need to serialize admin_ids or fetch admins for the
> group for this check to work? maybe we should serialize a property
> 'moreThanOneAdmin'
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2282/files#r34137079>.
>